### PR TITLE
Notification for wifi connections

### LINKED ISSUE #477: Add notification for WiFi connections
With the number of users over the limit.

### PR DESCRIPTION
**Pull request by luksan47**
Fixes #477

### CODE REVIEW

File: app/Models/WifiConnection.php [hunk 1]
```diff
@@ -48,4 +49,9 @@ public function getColor()
 
         return 'green';
     }
+
+    public static function notifications()
+    {
+        return User::role(Role::INTERNET_USER)->where('reachedWifiConnectionLimit', true)->count();
```
Comment (kdmnk) [inline app/Models/WifiConnection.php:55]: Got `Unknown column 'reachedWifiConnectionLimit' in 'where clause' (SQL: select count(*) as aggregate from `users` where exists (select * from `roles` inner join `role_users` on `roles`.`id` = `role_users`.`role_id` where `users`.`id` = `role_users`.`user_id` and `name` = internet-user) and `reachedWifiConnectionLimit` = 1)` for this line.

**Review by kdmnk — CHANGES_REQUESTED** [review]

File: app/Models/WifiConnection.php (hunk 1 above)
Comment (luksan47) [reply app/Models/WifiConnection.php:55]: Weird, worked for me, but not anymore. Let me check...

Comment (luksan47) [reply app/Models/WifiConnection.php]: Probably the cache stored the good results, and the query did not run. Now it should work.

**Review by kdmnk — APPROVED** [review]
LGTM

**Pull request merged**